bodhi/client/__init__.py

```diff
@@ -79,6 +79,8 @@ def new(user, password, **kwargs):
             print_resp(resp, client)
         except bindings.BodhiClientException as e:
             click.echo(str(e))
+	except Exception as e:
+	    click.echo("wrong password, please try again")
```

Comment:
If you put a ```print type(e)``` in this block and then type a wrong password, you will see what the specific Exception type is that gets raised on authentication error. That's the Exception you probably want to catch here instead of catching all Exceptions.